packages/styles/scss/patterns/sub-patterns/content-item/_content-item.scss

```diff
@@ -28,6 +28,10 @@
       color: $text-01;
       @include carbon--type-style('body-long-02', true);
     }
+
+    .#{$prefix}--link-with-icon__container {
+      margin-top: $carbon--spacing-05;
+    }
```

Comment:
What's the intent behind this here? I ask because I'm wondering if we should apply it somewhere else instead of inside of the `CTA` component if possible. If that's not going to work can we get a class name passed into it?

Reply:
It's still within the scope of `.#{$prefix}--content-item` so it's not going to affect the `CTA` component being used in other places right? I'm fine with having classname passed to it

Reply:
Yeah, I guess my concern is the `link-with-icon` specific call out here. What happens if in the future that name changes or we decide to allow cards here. Is that spacing then going to be applied still (assuming it's all still relevant of course)?

Reply:
wrapped in it a custom class